display_information/update-graphics/README.md

```diff
@@ -11,8 +11,8 @@ A field worker may want to update a graphic's properties to adjust which real-wo
 ## How to use the sample
 
 Update Graphic's Location:
-  - click on a graphic, then click the update location button
-  - now click a new location on the map to move graphic
+  - click on a graphic, then drag the graphic to a new location
+  - press ESCAPE to stop moving the graphic
```

Comment:
Technically escape doesn't stop the moving, it de-selects the graphic. Also, it's not just escape, but a single click outside the graphic will also de-select it.